docs/DeveloperGuide.md

```diff
@@ -391,31 +380,101 @@ For all use cases below, refer to the [User Guide](UserGuide.md) for more detail
+  - **1c1.** Same as 1a1.
+
 
-#### **Use Case: Add a Wedding**
+#### **Use Case: UC6 - Add a Wedding**
 
 **MSS:**
 
 1. Wedding Organizer adds a wedding using the addw command.
+2. Wedding details are added to the wedding list.
    Use case ends.
 
 **Extensions:**
 
 - **1a.** Contact name does not exist in the address book
     - **1a1.** Error message displayed in result display.
         - Use case ends.
-- **1b.** Contact/Wedding index is not in the displayed list
+  - **1b.** Contact/Wedding index is not in the displayed list
+      - **1b1.** Error message displayed in result display.
+          - Use case ends.
+  - **1c.** Duplicate contacts with name containing inputted name
+      - **1c1.** Bridal Boss filters the person list to show contacts containing inputted name.
+      - **1c2.** Use case resumes from step 1.
+    - **1d.** Contact is already a client of another wedding
+      - **1d1.** Error message displayed in result display.
+          - Use case ends.
+
+#### **Use Case: UC7 - View a Wedding**
+
+1. Wedding Organizer views a wedding using the vieww command.
+2. Wedding details are displayed.
+Use case ends.
+
+**Extensions:**
+
+- **1a.** No prefixes inputted.
+    - **1a1.** Error message displayed in result display.
+        - Use case ends.
+- **1b.** Wedding name does not exist in the address book
+    - **1b1.** Same as 1a1.
+- **1c.** Wedding index is not in the displayed list
+    - **1c1.** Same as 1a1.
+- **1d.** Duplicate wedding with name containing inputted name
+    - **1d1.** Bridal Boss filters the wedding list to show weddings containing inputted name.
+    - **1d2.** Same as 1a1.
+
+#### **Use Case: UC8 - Delete a wedding**
+1. Wedding Organizer deletes a wedding using the deletew command.
+2. Wedding is removed from the address book.
+Use case ends.
+
+**Extensions:**
+
+- **1a.** Wedding name does not exist in the address book
+    - **1a1.** Error message displayed in result display.
+        - Use case ends.
+- **1b.** Wedding index is not in the displayed list
     - **1b1.** Error message displayed in result display.
         - Use case ends.
-- **1c.** Duplicate contacts with name containing inputted name
-    - **1c1.** Bridal Boss filters the person list to show contacts containing inputted name.
-    - **1c2.** Use case resumes from step 1.
-  - **1d.** Contact is already a client of another wedding
-    - **1d1.** Error message displayed in result display.
+- **1c.** Duplicate wedding with name containing inputted name
+    - **1c1.** Same as 1d1 in <u>UC7 - View a wedding</u>
+
+#### **Use Case: UC9 - Edit a Wedding**
+
+1. Wedding Organizer edits a contact using the edit command.
+2. Contact details are updated.
+Use case ends.
+
+**Extensions:**
+
+- **1a.** Contact details is not valid
+    - **1a1.** Error message displayed in result display.
         - Use case ends.
+- **1b.** Contact index is not in the displayed list.
+  - **1b1.** Same as 1a1.
+- **1c.** Duplicate contacts.
```

Comment:
duplicate weddings? anyway editw is specified by index so shouldn't have duplicate cases i think